torchx/schedulers/docker_scheduler.py

```diff
@@ -359,6 +364,12 @@ def _run_opts(self) -> runopts:
             default=None,
             help="list of glob patterns of environment variables to copy if not set in AppDef. Ex: FOO_*",
         )
+        opts.add(
+            "env",
+            type_=Dict[str, str],
+            default=None,
+            help="environment varibles to be passed to the run (e.g. ENV1:v1,ENV2:v2,ENV3:v3)",
```

Comment:
can we document that either , or ; will work?

nit: spelling `variables`

Reply:
may also be good to document that this will apply on top of the copied envs

Reply:
Done